registry/src/options.rs

```diff
@@ -0,0 +1,28 @@
+// Copyright 2023 Helsing GmbH
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use clap::Parser;
+use std::net::SocketAddr;
+use url::Url;
+
+#[derive(Parser, Clone, Debug)]
+pub struct Options {
+    /// Address to listen to for incoming connections.
+    #[clap(long, short, env, default_value = "0.0.0.0:8000")]
```

Comment:
This would be neat, because we could have a standard port (not 8000) which we expect registries to run, allowing people to run a local registry proxy which does not interfere with their development setup (3000, 8000) or alike

Reply:
```
$ passgen "[3-9][0-9]{3}"
4367
```

How about this one?